ingest/importer/spreadsheet/ingest_workbook.py

```diff
@@ -17,6 +18,14 @@ def from_file(cls, file_path, read_only=True) -> 'IngestWorkbook':
         workbook = load_workbook(filename=file_path, read_only=read_only)
         return cls(workbook)
 
+    def is_large(self):
```

Comment:
just a nitpick, could be `exceeds_row_limit` instead of is_large but doesn't really matter lol

Reply:
`exceeds_row_limit(row_limit_from_env_var)`

Reply:
the function name has changed to reflect that it now returns the list of offending sheets exceeding row limit.